JSDemos/Demos/Toolbar/Multiline/Angular/app/app.component.css

```diff
@@ -0,0 +1,61 @@
+::ng-deep .dx-resizable-handle::after {
+  content: "";
+  background-color: #337ab7;
+  position: absolute;
+  width: 12px;
+  height: 12px;
+  border: 1px solid #337ab7;
+  top: calc(50% - 7px);
```

Comment:
Can we avoid using calc in CSS?
It's not very good for performance
Maybe we can resolve some cases where we use calc with div containers with margins/paddings?

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/cdcf8a764e3c2f7c237fdd5b57109941e8a6bd8c